components/vars/css/globals/spectrum-dimensionAliases.css

```diff
@@ -18,22 +18,42 @@
   --spectrum-alias-single-line-height: var(--spectrum-global-dimension-size-400);
   --spectrum-alias-single-line-width: var(--spectrum-global-dimension-size-2400);
   --spectrum-alias-workflow-icon-size: var(--spectrum-global-dimension-size-225);
-  --spectrum-alias-heading-han-display1-text-size: var(--spectrum-global-dimension-font-size-1000);
-  --spectrum-alias-heading-han-xxxl-text-size: var(--spectrum-global-dimension-font-size-1000);
-  --spectrum-alias-heading-han-display1-margin-top: var(--spectrum-global-dimension-font-size-900);
-  --spectrum-alias-heading-han-xxxl-margin-top: var(--spectrum-global-dimension-font-size-900);
+  --spectrum-alias-heading-display1-text-size: var(--spectrum-global-dimension-font-size-1300);
```

Comment:
@bernhard-adobe `--spectrum-alias-heading-display1-text-size` is present here, it seems it was moved from `spectrum-medium.css` to `spectrum-dimensionAliases.css`. The variable is present on the dev site and works, but its value is now `--spectrum-global-dimension-font-size-1300`, whereas previously it was `--spectrum-global-dimension-font-size-1100`. for medium and `--spectrum-global-dimension-font-size-1000` for large.

@NateBaldwinDesign, was this intended?

Reply:
Yes this was an intentional increase in size for the larger type styles

Reply:
Thanks @lazd @NateBaldwinDesign. I manually reverted the change in `spectrum-dimensionAliases.css` and restored it with the old value in `spectrum-medium.css`. Here is the VR result: https://spectrum-visual-regression.ci.corp.adobe.com/job/spectrum-css-vr-test-asset/197/artifact/spectrum-css/backstop_data/html_report/index.html

It seems like all the 1px offset issue gone. So the question now would be what should we do ?

Reply:
Ok, so now that the sizes are the same between medium and large, the alias is no longer present in `spectrum-medium.css` and `spectrum-large.css`, and instead is moved up into `spectrum-dimensionAliases.css`. These variables are definitely not missing, though the change in their value could be related to the 1px shift we're seeing. @bernhard-adobe hope this helps.